Breaking tests in CI

## Is your feature request related to a problem? Please describe

Our CI reports a failure when trying to run"integration" tests.

{"lvl":"info","host":"2ec8dafb2a8d","srv":"test","ver":"1.0.0","time":"2019-02-25T07:03:50.661389192Z","src":"http/component.go:98","msg":"HTTP component listening on port 50000"}
--- FAIL: TestServer_Run_Shutdown (0.21s)
    --- FAIL: TestServer_Run_Shutdown/success (0.10s)
        <autogenerated>:1: 
            	Error Trace:	service_test.go:58
            	Error:      	Received unexpected error:
            	            	listen tcp :50000: bind: address already in use
            	Test:       	TestServer_Run_Shutdown/success
FAIL
coverage: 89.3% of statements

## Describe the solution

It seems that a previous test service was not shutdown completely. We could run every test on another port to avoid port collission.

--- a/service_test.go
+++ b/service_test.go
@@ -3,7 +3,6 @@ package patron
 import (
 	"context"
 	"os"
-	"strconv"
 	"testing"
 
 	"github.com/stretchr/testify/assert"
@@ -42,22 +41,21 @@ func TestNewServer(t *testing.T) {
 
 func TestServer_Run_Shutdown(t *testing.T) {
 	tests := []struct {
-		name       string
-		cp         Component
-		port       int64
-		wantRunErr bool
+		name    string
+		cp      Component
+		port    string
+		wantErr bool
 	}{
-		{name: "success", cp: &testComponent{}, port: 50000, wantRunErr: false},
-		{name: "failed to run", cp: &testComponent{errorRunning: true}, port: 50001, wantRunErr: true},
+		{name: "success", cp: &testComponent{}, port: "50000", wantErr: false},
+		{name: "failed to run", cp: &testComponent{errorRunning: true}, port: "50001", wantErr: true},
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			err := os.Setenv("PATRON_HTTP_DEFAULT_PORT", strconv.FormatInt(tt.port, 10))
-			assert.NoError(t, err)
+			os.Setenv("PATRON_HTTP_DEFAULT_PORT", tt.port)
 			s, err := New("test", "", Components(tt.cp, tt.cp, tt.cp))
 			assert.NoError(t, err)
 			err = s.Run()
-			if tt.wantRunErr {
+			if tt.wantErr {
 				assert.Error(t, err)
 			} else {
 				assert.NoError(t, err)

